src/pages/Team.vue

```diff
@@ -0,0 +1,346 @@
+<template>
+  <Layout page="team">
+    <div class="container relative mx-auto">
+      <div class="items-center flex flex-wrap">
+        <div class="w-full lg:w-6/12 px-4 ml-auto mr-auto text-center">
+          <div class="py-4">
+            <h1 class="font-semibold text-5xl underline">the craftsmen</h1>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="p-8 flex items-center justify-center people-card-section">
+      <div
+        class="
+          grid grid-cols-1
+          sm:grid-cols-2
+          md:grid-cols-2
+          xl:grid-cols-3
+          cursor-pointer
+          gap-6
+        "
+      >
+        <div
+          class="shadow-xl transform transition duration-500 hover:scale-105"
+          @click="scrollMeTo('joinus')"
+        >
+          <div
+            class="
+              flex flex-col
+              items-center
+              justify-center
+              bg-white
+              p-4
+              shadow
+              rounded-lg
+              height-100
+            "
+          >
+            <div
+              class="
+                inline-flex
+                shadow-lg
+                border border-gray-200
+                rounded-full
+                overflow-hidden
+                h-40
+                w-40
+              "
+            >
+              <img src="/you.png" alt="" class="h-full w-full" />
+            </div>
+
+            <h2 class="mt-4 font-bold text-xl underline">you ?</h2>
+
+            <p class="text-xs text-gray-500 text-center mt-3 ellipsis">
+              quote that defines you?
+            </p>
+          </div>
+        </div>
+        <div
```

Comment:
We should make this a common component as code is the same for members & founders. It just differs at what kind of data we're sending as props.

Reply:
PR updated as per your feedback.
Please review & resolve this conversation.